data_steward/tools/create_tier.py

```diff
@@ -9,12 +9,39 @@
 from utils import auth
 from utils import bq
 from utils import pipeline_logging
+from cdr_cleaner import clean_cdr
+from tools.snapshot_by_query import create_schemaed_snapshot_dataset
 from constants.cdr_cleaner import clean_cdr as consts
 
 LOGGER = logging.getLogger(__name__)
 
 TIER_LIST = ['controlled', 'registered']
 DEID_STAGE_LIST = ['deid', 'base', 'clean']
```

Comment:
We need to update this line to:
```suggestion
DEID_STAGE_LIST = ['deid', 'deid_base', 'deid_clean']
```
Otherwise, it doesn't create the correct key for the rules dictionary.